mkosi/__init__.py

```diff
@@ -3090,6 +3178,20 @@ def build_image(context: Context) -> None:
             tools=context.config.tools(),
             sandbox=context.sandbox,
         )
+    elif context.config.output_format == OutputFormat.oci:
+        make_tar(
+            context.root, context.staging / "rootfs.layer",
+            tools=context.config.tools(),
+            sandbox=context.sandbox,
+            # We do not want SELinux xattrs in OCI layers
+            # as they seldom make sense inside containers.
+            options=["--xattrs-exclude", "security.selinux"],
```

Comment:
People make VMs now from OCI images, so I think excluding selinux is a bit presumptuous. I would prefer the selinux xattrs are stripped out when extracting the OCI image, not when creating it.

Reply:
umoci logged warnings when extracting an OCI with selinux labels but I can keep them in if that is preferred